invokeai/app/invocations/flux_lora_loader.py

```diff
@@ -155,19 +167,13 @@ def invoke(self, context: InvocationContext) -> FluxLoRALoaderOutput:
 
             added_loras.append(lora.lora.key)
 
-            if self.transformer is not None:
-                if output.transformer is None:
-                    output.transformer = self.transformer.model_copy(deep=True)
+            if self.transformer is not None and output.transformer is not None:
                 output.transformer.loras.append(lora)
 
-            if self.clip is not None:
-                if output.clip is None:
-                    output.clip = self.clip.model_copy(deep=True)
+            if self.clip is not None and output.clip is not None:
                 output.clip.loras.append(lora)
 
-            if self.t5_encoder is not None:
-                if output.t5_encoder is None:
-                    output.t5_encoder = self.t5_encoder.model_copy(deep=True)
+            if self.t5_encoder is not None and output.t5_encoder is None:
```

Comment:
I think this may be a typo

```suggestion
            if self.t5_encoder is not None and output.t5_encoder is not None:
```

Reply:
Good catch it was a typo